include/deal.II/multigrid/mg_transfer_internal.h

```diff
@@ -137,6 +137,52 @@ namespace internal
       MGLevelObject<std::shared_ptr<const Utilities::MPI::Partitioner>>
         &vector_partitioners);
 
+
+
+    /**
+     * Helper function for setup_transfer. Checks for identity constrained
+     * dofs and replace with the indices of the dofs to which they are
+     * constrained
+     */
+    void
+    resolve_identity_constraints(
+      const MGConstrainedDoFs *             mg_constrained_dofs,
+      const unsigned int                    level,
+      std::vector<types::global_dof_index> &dof_indices);
+
+
+
+#ifndef DOXYGEN
+
+    /* ----------------------- Inline functions
+     * --------------------------------- */
+
+
+    inline void
+    resolve_identity_constraints(
+      const MGConstrainedDoFs *             mg_constrained_dofs,
+      const unsigned int                    level,
+      std::vector<types::global_dof_index> &dof_indices)
+    {
```

Comment:
Since this function is not a template, I wonder whether we can leave this in the .cc file? (But obviously without the anonymous namespace.)